### ISSUE TITLE
loss clsoe to -1 at the begining of training

### ISSUE THREAD

**Issue by JudasDie** [PatrickHua/SimSiam#6]
Has anyone met the problem that the loss close to -1 at the beginning of training? BTW, the training data is not sourced from traditional classification data like Imagenet or cifar.

**Comment by PatrickHua** [comment]
What kind of data? cifar/imagenet put target object in the middle and only one instance may appear for contrastive learning. The data augmentation (resized random crop etc.) is tailored to this type of input.

**Comment by JudasDie** [comment]
> What kind of data? cifar/imagenet put target object in the middle and only one instance may appear for contrastive learning. The data augmentation (resized random crop etc.) is tailored to this type of input.

Similar to coco format, yet I crop the target from the original image. The target is centrally present in the image, as in Imagenet.

**Comment by matthiasware** [comment]
> Has anyone met the problem that the loss close to -1 at the beginning of training? BTW, the training data is not sourced from traditional classification data like Imagenet or cifar.

Yes! Especially for parameters that differ from the ones given in the paper!
Checkout Fig.2 in the paper and track the channel std!

**Comment by JudasDie** [comment]
> > Has anyone met the problem that the loss close to -1 at the beginning of training? BTW, the training data is not sourced from traditional classification data like Imagenet or cifar.
> 
> Yes! Especially for parameters that differ from the ones given in the paper!
> Checkout Fig.2 in the paper and track the channel std!

Hi，do you use Imagenet or custom data?

**Comment by matthiasware** [comment]
> > > Has anyone met the problem that the loss close to -1 at the beginning of training? BTW, the training data is not sourced from traditional classification data like Imagenet or cifar.
> > 
> > 
> > Yes! Especially for parameters that differ from the ones given in the paper!
> > Checkout Fig.2 in the paper and track the channel std!
> 
> Hi，do you use Imagenet or custom data?

custom dataset!
but i can get good results by choosing the right parameters!

**Comment by JudasDie** [comment]
> > > > Has anyone met the problem that the loss close to -1 at the beginning of training? BTW, the training data is not sourced from traditional classification data like Imagenet or cifar.
> > > 
> > > 
> > > Yes! Especially for parameters that differ from the ones given in the paper!
> > > Checkout Fig.2 in the paper and track the channel std!
> > 
> > 
> > Hi，do you use Imagenet or custom data?
> 
> custom dataset!
> but i can get good results by choosing the right parameters!

Could you share some experience? e.g., how to choose the parameter? or which parameter affects most?

**Comment by PatrickHua** [comment]
> > Has anyone met the problem that the loss close to -1 at the beginning of training? BTW, the training data is not sourced from traditional classification data like Imagenet or cifar.
> 
> Yes! Especially for parameters that differ from the ones given in the paper!

Interesting! So SimSiam is very sensitive to certain parameters right? What hyperparameter did you change ?

> Checkout Fig.2 in the paper and track the channel std!

What's with the channel std?
